config/initializers/devise.rb

```diff
@@ -251,7 +251,7 @@
   # config.navigational_formats = ['*/*', :html]
 
   # The default HTTP method used to sign out a resource. Default is :delete.
-  config.sign_out_via = :delete
+  config.sign_out_via = :get
```

Comment:
I’m not sure about having sign out as a GET action. GET action are supposed to not change data, which a logout does. Also brings problems with URL prefetching (GET URLs should be able to be prefetched, which also is not the case with a logout). What do you think?

Reply:
Yea, my preference is definitely to keep it as a delete action. I'll play with it again and see if I can get it to work as the appropriate action.